pages/hud.coffee

```diff
@@ -1,3 +1,21 @@
+document.addEventListener "keydown", (event) ->
```

Comment:
Why isn't this part of the `keydown` handler for FindMode?

Also, why are we only handling `keydown`, here?  Will that not cause problems for non-Latin keyboards?